### PR TITLE
Add auto rewind documentation to playback page

### PR DESCRIPTION
**Pull request by Matth7878**
Automatic rewind documentation on "Playback" page.
Let me know if there is something wrong as I am note used to contribution on GitHub. :-/

### CODE REVIEW

File: _documentation/playback/auto_rewind.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Automatic rewind
+layout: doc
+level: "2"
+group: "playback"
+icon: "fas fa-keyboard"
```
Comment (ByteHamster) [inline _documentation/playback/auto_rewind.md:6]: I think the `fa-backward` icon would fit better here.

```suggestion
icon: "fas fa-backward"
```

File: _documentation/playback/auto_rewind.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+# Automatic rewind after starting an episode which has been paused
+
+When an episode has been paused and is played again AntennaPod will automatically rewind.
+
+How much is rewinded is not configurable and depends on how long an episode has been paused.
+
+Rewinds timing are currently :
```
Comment (ByteHamster) [inline _documentation/playback/auto_rewind.md:15]: I added a few words about *why* we rewind (also, removed the line breaks). Let me know what you think :)

```suggestion
When an episode has been paused and is played again, AntennaPod will automatically rewind, so it is easier to continue following the show. How much is rewound is not configurable and depends on how long an episode has been paused. Rewind timing are currently:
```

**Comment by Matth7878** [comment]
I am ok with what you told.
My bad for the icon, I wanted to change it but forgot afterward ! :-/

**Comment by Matth7878** [comment]
I guess I am supposed to merge / squash commit to have only one ? But I am looking how to do it ??? (I made change on web GitHub)

**Comment by ByteHamster** [comment]
I just squashed it (one can select different methods for merging). Should be available on the website in a few minutes :)

**Pull request merged**